File: oner-php/app/Supplier.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class Supplier extends Model 
{    
    protected $table = 'supplier';
    const CREATED_AT = 'created_time';
    const UPDATED_AT = 'updated_time';
    // public $timestamps = false;
    
    /**
     * The attributes that are mass assignable.
     *
     * @var array
     */
    protected $fillable = [
        'dealer',
        'company',
        'credit_term',
        'contact',
        'address',
        'remark',
        'created_user_id',
        'updated_user_id',
    ];

}

File: oner-python/src/app/infrastructure/models/Inventory/supplier.py
from enum import auto
from sqlalchemy.sql.sqltypes import String
from sqlalchemy import Column
from app.infrastructure.models import BaseModel

class Supplier(BaseModel):
    __tablename__='supplier'

    dealer=Column(String(32))
    company=Column(String(32))
    credit_term=Column(String(10))
    address=Column(String)
    contact=Column(String)
    remark=Column(String)